pkg/encoding/v1/encoding_test.go

```diff
@@ -545,12 +549,12 @@ services:
 - name: frontend
   containers:
   - image: tomaskral/kompose-demo-frontend:test
-	env:
-	- KEY=value
-	- KEY2=value2
-	ports:
-	- port: 5000:80
-	- port: 5001:81
+    env:
+    - KEY=value
+    - KEY2=value2
+    ports:
+    - port: 5000:80
+    - port: 5001:81
```

Comment:
this was failing on the error that there was `tab` defined and not space, so but it was expected to fail on `EXCESSKEY` so this was discovered using this PR.